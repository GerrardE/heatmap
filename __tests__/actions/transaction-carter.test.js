import toJson from 'enzyme-to-json';
import TransactionCarter from '../../src/actions/transactions-carter.json';

const setupCarter = () => {
  const wrapper = TransactionCarter;
  return wrapper
}

describe('TransactionCarter', () => {
  test('should render the TransactionCarter properly', () => {
    let wrapperCarter;
    wrapperCarter = setupCarter();
    expect(toJson(wrapperCarter)).toMatchSnapshot();
  })
});
