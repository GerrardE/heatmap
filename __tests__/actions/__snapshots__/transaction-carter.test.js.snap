// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransactionCarter should render the TransactionCarter properly 1`] = `undefined`;
